### PR TITLE
Feature/progot

### PR DESCRIPTION
**Pull request by michalk8**
TODOs:
- [x] polish the API
- [x] docstrings
- [x] polish the tutorial (almost done)
- [ ] tests

### CODE REVIEW

**Review by marcocuturi — APPROVED** [review]
this all looks good to me

File: docs/tutorials/linear/700_progot.ipynb [hunk 1]
```diff
@@ -0,0 +1,618 @@
+{
```
Comment (marcocuturi) [inline docs/tutorials/linear/700_progot.ipynb:1]: <p>i would replace the "(dual of the entropic)" mention, as ProgOT doesn't return currently dual solutions, and, to some extent, is not really the EOT problem either.</p><p> </p><p>For instance:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">&nbsp;ProgOT is a progressive algorithm that outputs approximate solutions to the OT problem, returning both, a coupling matrix ... that aligns the source and target datasets (primal OT problem), and a map estimator (valid out of sample) ... that pushes-forward efficiently points from the source to the target.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/567/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (michalk8) [reply docs/tutorials/linear/700_progot.ipynb:1]: Done, thanks!

**Review by pkassraie — COMMENTED** [review]
Looks good to me!

**Pull request merged**